tests/lib/rules/lines-around-comment.js

```diff
@@ -1414,6 +1445,94 @@ ruleTester.run("lines-around-comment", rule, {
             }],
             parserOptions: { ecmaVersion: 6 },
             errors: [{ message: afterMessage, type: "Block", line: 4 }]
+        },
+
+        // ignorePattern
+        {
+            code:
+            "foo;\n\n" +
+            "/* eslint-disable no-underscore-dangle */\n\n" +
+            "this._values = values;\n" +
+            "this._values2 = true;\n" +
+            "/* eslint-enable no-underscore-dangle */\n" +
+            "bar",
+            options: [{
+                beforeBlockComment: true,
+                afterBlockComment: true,
+                applyDefaultIgnorePatterns: false
+            }],
+            errors: [
+                { message: beforeMessage, type: "Block", line: 7 },
+                { message: afterMessage, type: "Block", line: 7 }
+            ]
+        },
+        {
+            code: "foo;\n/* eslint */",
+            output: "foo;\n\n/* eslint */",
```

Comment:
Now that https://github.com/eslint/eslint/commit/18a519fa2971e54a639e2b3320aeecd9a31b390a is merged, can you replace these with `output: null`?